### DISCUSSION TITLE
DynamicRangeSlider showing different min/max values

### DISCUSSION THREAD

**Discussion by schellerg** [appbaseio/reactivesearch#2018] [Q&A]
Hi,

I'm having an issue with the DynamicRangeSlider component.
I´m using it as a price filter in my project and (I suppose) its returning the min/max price of all products instead of the ones within the current subset.

For example: if I´m on a page about books (my current subset); the cheapest book costs 5 and the most expensive costs 15. My dynamic range is showing min/max of 2 and 99, respectively. So, it´s like it isn't computing the prices relative to the current books.

I´ve already tried some things, like the `react` prop in order to "connect" or "relate" the price range with the list, to no success.
There´s some way to debug the dataset that the DynamicRangeSlider is using?

**Comment by siddharthlatest** [comment]
@schellerg DynamicRangeSlider doesn't work based on the current page's value, but the values that are min/max for its own query.

If you can share a reproducible CodeSandbox example and/or share a network request just for the RangeSlider's query that produces this output, that would help to tell more.

**Reply by schellerg** [reply]
Hi, @siddharthlatest

I can share a network request and if necessary, the code of the provider and the range slider component.
Here is the payload:
```
{
    "preference": "PriceSensor__range__internal"
}
{
    "query": {
        "match_all": {}
    },
    "size": 30,
    "_source": {
        "includes": [
            "*"
        ],
        "excludes": []
    },
    "from": 0,
    "sort": [
        {
            "price": {
                "order": "desc"
            }
        }
    ],
    "aggs": {
        "min": {
            "min": {
                "field": "price"
            }
        },
        "max": {
            "max": {
                "field": "price"
            }
        }
    }
}

{
    "preference": "PriceSensor"
}
{
    "query": {
        "match_all": {}
    },
    "size": 30,
    "_source": {
        "includes": [
            "*"
        ],
        "excludes": []
    },
    "from": 0,
    "sort": [
        {
            "price": {
                "order": "desc"
            }
        }
    ]
}
```

I´ve noticed that there are more than one request, as we can see on the console:

![image](https://user-images.githubusercontent.com/26773430/177850935-d6d33b5d-cfbd-4771-aed7-b88d07bd1598.png)

Tell me if you need more info about the request so I can provide you.
